website/addons/onedrive/serializer.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from website.addons.base.serializer import OAuthAddonSerializer
+
+from website.util import api_url_for, web_url_for
+
+logger = logging.getLogger(__name__)
+
+logging.getLogger('onedrive1').setLevel(logging.WARNING)
+
+
+class OneDriveSerializer(OAuthAddonSerializer):
```

Comment:
I think you will need to add some more fields to this class in order for it to be instantiated. See the `abc.abstractproperty`s defined in https://github.com/CenterForOpenScience/osf.io/blob/develop/website/addons/base/serializer.py#L15. 

E.g.
`addon_short_name` = 'onedrive'